snuba/datasets/events_processor_base.py

```diff
@@ -228,7 +234,7 @@ def process_insert(
         self.extract_common(processed, event, metadata)
         self.extract_custom(processed, event, metadata)
 
-        sdk = data.get("sdk", None) or {}
+        sdk = data.get("sdk", None) or {}  # type: ignore
```

Comment:
What was the issue here ?

Reply:
"sdk" does not exists on the data object.